### PR TITLE
MEN-5948

### PR DESCRIPTION
**Pull request by mzedel**
this needs:
- https://github.com/mendersoftware/tenantadm/pull/411
- https://github.com/mendersoftware/workflows-enterprise/pull/464
- https://github.com/mendersoftware/useradm-enterprise/pull/578
- https://github.com/mendersoftware/integration/pull/2233

Signed-off-by: Manuel Zedel <manuel.zedel@northern.tech>

### CODE REVIEW

**Review by tranchitella — APPROVED** [review]
I did not review the code, only tested the functionality in combination with the backend.

File: src/js/components/footer.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    alignItems: 'center'
+  }
+}));
+
+export const Footer = () => {
+  const { classes } = useStyles();
+  return (
+    <div className={classes.footer}>
+      <b>© 2022 Northern.tech AS</b>
+      <b>Terms of service</b>
```
Comment (michaelatmender) [inline src/js/components/footer.js:23]: These should be links :)

File: src/js/components/login/signup-steps/orgdata-entry.js [hunk 2]
```diff
@@ -1,74 +1,115 @@
...
+            value={name}
+            validations="isLength:1"
+          />
+          {!emailVerified && <TextInput hint="Email *" label="Email *" id="email" required={true} validations="isLength:1,isEmail" value={email} />}
+
+          <div className="flexbox center-aligned margin-top slightly-smaller">
+            <p className="margin-bottom-none margin-top-none muted" style={{ marginRight: 4 }}>
+              Choose which instance to set up your Mender account.
+            </p>
+            <a href="https://mender.io/plans" target="_blank" rel="noopener noreferrer">
```
Comment (michaelatmender) [inline src/js/components/login/signup-steps/orgdata-entry.js:52]: Switch url to the page currently at https://docs.mender.io/general/hosted%20mender%20regions

But the URL needs to be fixed first

File: src/js/components/footer.js (hunk 1 above)
Comment (mzedel) [reply src/js/components/footer.js:23]: argh, sorry I missed this! https://github.com/mendersoftware/gui/pull/3315 should bring them in 👌

File: src/js/components/login/signup-steps/orgdata-entry.js (hunk 2 above)
Comment (mzedel) [reply src/js/components/login/signup-steps/orgdata-entry.js:52]: https://github.com/mendersoftware/gui/pull/3316 to be merged once the docs state is ready...

**Pull request merged**